go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -2,6 +2,7 @@ package services
 
 import (
 	"fmt"
```

Comment:
General question concerning this commit and maybe overall for this patch: how does it handle downgrades? We are changing the service implementation when upgrading to the version containing this patch, are we certain we clean all of that (i.e: dedicated worker load balancers) up if we downgrade back again? Or do we say: we can't, but there should be no impact on networking if we still have that data lying around?

Reply:
I'll handle downgrade in a follow up. Le'ts leave this comment unresolved so I remember to do so.